pupa/scrape/schemas/vote.py

```diff
@@ -7,7 +7,7 @@
     "properties": {
         'identifier': {"type": "string", "blank": True,},
         'motion_text': {"type": "string" },
-        'motion_classification': {"items": {"type": "string", "enum": common.VOTE_CLASSIFICATIONS},
+        'motion_classification': {"items": {"type": "string"},
```

Comment:
Well, I assumed we'd just add to that set as time goes on, let me think it over.